apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx

```diff
@@ -0,0 +1,26 @@
+import Stand from "./Stand";
+import stand1 from "./stand-1.png";
+import stand2 from "./stand-2.png";
+
+const MentorVolunteer = () => {
+	return (
+		<div className="grid max-w-lg lg:max-w-none lg:grid-cols-[repeat(2,450px)] lg:items-stretch justify-center m-auto gap-x-8 gap-y-12 mt-24">
+			<Stand
+				imageSrc={stand1}
+				header="Apply to be a Mentor"
+				description="Interested in inspiring the next generation of developers? Our mentors are vital to making HackUCI 2023 successful and enjoyable for our hackers."
```

Comment:
Issue (here and later in the file): "HackUCI 2023" should be "IrvineHacks 2024".